index.html

```diff
@@ -14,16 +14,13 @@
         <h3>Interested in Joining PiE?</h3>
         <p>Fill out our interest form so we can learn about you, and you can get the latest information from us!</p>
         <p>You can also follow us on <a href="https://facebook.com/pierobotics">Facebook</a> to see our events this semester.</p>
-        <a class="btn btn-primary" href="https://docs.google.com/forms/d/e/1FAIpQLSfrJ_ImQvy6-D_70E5-bIJFSt8hr10dZzpZlRTBDvcB5vGfMg/viewform" role="button">Interest Form</a>
+        <a class="btn btn-primary" target="_blank" href="https://docs.google.com/forms/d/e/1FAIpQLSfrJ_ImQvy6-D_70E5-bIJFSt8hr10dZzpZlRTBDvcB5vGfMg/viewform" role="button">Interest Form</a>
         </div>
       <div class="col-md-6">
-        <h3>Recruiting Schedule</h3>
-        <p>
-        <strong>Game Night - 09/08/2017</strong>: in 540 Cory, from 6:00 - 8:00 PM! <br>
-        <strong>Infosession #2 - 09/11/2017</strong>: in 405 Soda, from 7:00 - 9:00 PM! <br>
-        <strong>First Workssion and Project Expo - 09/13/2017</strong>: in 212 O'Brien, from 7:30 - 9:00 PM! <br>
-        </p>
-        <a class="btn btn-info" href="../get-involved/" role="button">More Staff info!</a>
+        <h3>PiE on the Road</h3>
+        <p>PiE on the Road is a traveling workshop developed by Pioneers in Engineering and hosted at local high schools to bring a new STEM experience to a high school near you.</p>
+        <img src="assets/events/gearworkshopflyer_final.jpg" width="300">
```

Comment:
The button looks kind of strange. Can you write in the paragraph text "Learn More" and make that a hyperlink, and also link the image?